runhealthcheck.sh

```diff
@@ -13,9 +13,9 @@ while true ; do
        HTTPDPID=$(pidof -s httpd)
        fuser $2/tcp |& grep -w "$HTTPDPID"
 
-    elif [ $1 = "dnsmasq" ] ; then
-       DNSMASQPID=$(pidof dnsmasq)
-       fuser 67/udp |& grep -w "$DNSMASQPID"
+    #elif [ $1 = "dnsmasq" ] ; then
+    #   DNSMASQPID=$(pidof dnsmasq)
+    #   fuser 67/udp |& grep -w "$DNSMASQPID"
```

Comment:
Note to myself, FIXME: dhpc6 listens on different port

Reply:
547?